_extras/checkout.md

```diff
@@ -105,6 +102,21 @@ The lessons you may submit to are:
 3.  [Stable Software Carpentry lessons](https://software-carpentry.org/lessons/):
     we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives.
 
+> ## What Makes a Good Exercise?
+>
+> 1.  Follow the coding style of the lesson you're submitting to.
+>     For example,
+>     [our SQL lesson][swc-sql] capitalizes keywords but uses lower case for table and field names.
+> 2.  Include a solution to your exercise.
+> 3.  Make sure the exercise runs on Windows, Mac OS X, and Linux,
+>     since most workshops have users on all three platforms.
```

Comment:
On Mon, May 02, 2016 at 02:47:35PM -0700, Greg Wilson wrote:
> > +> 3.  Make sure the exercise runs on Windows, Mac OS X, and Linux,
> > +>     since most workshops have users on all three platforms.
> 
> Fresh wording pushed - please have a look.

“Do not rely…” wording from 1ff5171 looks good to me.